### PR TITLE
Feature/index view and login

### PR DESCRIPTION
**Pull request by rilindpozhegu**
Krijimi i faqes se pare te forumit, pjesa e dyte eshte Login dhe se fundmi pjesa e regjistrimit Signup.

### CODE REVIEW

File: src/main/resources/templates/Register.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Register</title>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```
Comment (flakron) [inline src/main/resources/templates/Register.html]: script tag-at shkojn ne fund te body-s, jo ne head

File: src/main/resources/templates/Register.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <nav class="navbar navbar-default container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="http://localhost:12/forum/src/main/resources/templates/">
+                <div class="nav-logo">
+                    <img src="http://www.clker.com/cliparts/1/d/7/e/12570918991185627521Ramiras_Earth_small_icon.svg.med.png">
+                </div>
+            </a>
+        </div>
+    </nav>
+    <div class="body1 container">
```
Comment (flakron) [inline src/main/resources/templates/Register.html]: qka osht ```body``` ?

File: src/main/resources/templates/index.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
+body {
```
Comment (flakron) [inline src/main/resources/templates/index.css]: nuk i ke perfille hiq rregullat qe shtrohen ne dokumentet qe ja kom dergu @GranitSahatqija 
kapi lexoj niher mire qka po shkrun edhe aplikoj ktu

File: src/main/resources/templates/Register.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```
Comment (flakron) [inline src/main/resources/templates/Register.html]: nuk i ke perfille hiq rregullat qe shtrohen ne dokumentet qe ja kom dergu @GranitSahatqija
kapi lexoj niher mire qka po shkrun edhe aplikoj ktu

File: src/main/resources/templates/Register.html (hunk 1 above)
Comment (rilindpozhegu) [inline src/main/resources/templates/Register.html]: Okej i rregulloj

File: src/main/resources/templates/.picasa.ini [hunk 5]
```diff
@@ -0,0 +1,2 @@
+[cover-photo-2.jpg]
+backuphash=27810
```
Comment (GranitSahatqija) [inline src/main/resources/templates/.picasa.ini]: shka o qekjo

File: src/main/resources/templates/.picasaoriginals/.picasa.ini [hunk 6]
```diff
@@ -0,0 +1,12 @@
+[cover-photo-2.jpg]
```
Comment (GranitSahatqija) [inline src/main/resources/templates/.picasaoriginals/.picasa.ini]: shka o qekjo

**Comment by GranitSahatqija** [comment]
kqyri qeto foto qe i ki bo push, 3 her osht njona

File: src/main/resources/templates/.picasaoriginals/.picasa.ini (hunk 6 above)
Comment (rilindpozhegu) [inline src/main/resources/templates/.picasaoriginals/.picasa.ini]: fotografi e emertune qeshtut

Comment (rilindpozhegu) [inline src/main/resources/templates/.picasaoriginals/.picasa.ini]: nenkupton e dyta me renditje

File: src/main/resources/templates/index.css [hunk 7]
```diff
@@ -0,0 +1,467 @@
...
+        margin-left: 0;
+        left: 15px;
+        top: 2px;
+    }
+}
+/*------------------------------------------------------*\
+  Butoni shto ku ne te cilin ne mund te shtojme Topic's
+\*------------------------------------------------------*/
+
+.Shto-button {
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: klasat edhe id n css (id hiq s preferohen) duhen lowercase mu kon

File: src/main/resources/templates/index.css [hunk 8]
```diff
@@ -0,0 +1,467 @@
...
+    width: 300px;
+    height: 40px;
+    background-color: #43707B;
+    color: white;
+    font-weight: bold;
+    margin-left: 100px;
+    margin-top: 0px;
+}
+.Shto-button:hover {
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: plot paske klasa me shkronja t mdhaja, boni t vogla krejt, spe komentoj n tonat, ndreqi vet

File: src/main/resources/templates/index.html [hunk 9]
```diff
@@ -1,10 +1,388 @@
...
+        <nav class="navbar navbar-default ">
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a href="http://localhost/forum/src/main/resources/templates/">
+                    <img class="logo-foum" src="logo-forum.png">
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.html]: logo-forum, edhe n css njejt e ki

File: src/main/resources/templates/index.css [hunk 10]
```diff
@@ -0,0 +1,467 @@
...
+#social-tw:hover {
+    color: #4099FF;
+}
+#social-gp:hover {
+    color: #d34836;
+}
+#social-em:hover {
+    color: #f39c12;
+}
+.center-block {
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: qet klas nuk e shfrytzon kerka

File: src/main/resources/templates/index.css [hunk 11]
```diff
@@ -0,0 +1,467 @@
...
+    font-weight: bold;
+    outline: none;
+}
+.btn-reply:focus {
+    border: none;
+}
+.btn-reply:active {
+    border: none;
+}
+.slider-reply {
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: 2 her e ki qeto

File: src/main/resources/templates/index.html [hunk 12]
```diff
@@ -1,10 +1,388 @@
...
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <!-- Pjesa e panelit -->
+        <div class="panel panel-default">
+            <div class='panel-heading'>
+                <!-- Hederi i panelit -->
+                <div class='pull-left'>
+                    <a href="#"><img class="img-responsive img-rounded photo-user" src="skenderbeu.jpg"> </a>
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.html]: ndreqi elementet qe jon nested n krejt fajllin, se t ikin muhalife qe mujin me t qit probleme n layout. qe qetu psh  tagin se ki mshel

File: src/main/resources/templates/index.html [hunk 13]
```diff
@@ -1,10 +1,388 @@
...
+                    <div class='col-xs-12 col-sm-10 col-md-10 col-lg-10'>
+                        <div class="post-time">
+                            <p><small>Posted <abbr class='atooltip' data-toggle='tooltip' data-placement='top'>2 min. ago</abbr></small>
+                            </p>
+                        </div>
+                        <div class="title-of-panel">Hi</div>
+                        <br /> I would like to be able to display Accu-weather, or maybe Add Words in panels on my site. I was thinking this would just be a case of pasting the HTML in to a new custom panel..
+                        <br /> Wrong! can anyone please tell me how to do this?
+                        <br />
+                        <br /> <u>thanks</u>
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.html]: qeto <u> nuk jon t mira hiq, e shtijin userin me menu qe osht link

File: src/main/resources/templates/index.html [hunk 14]
```diff
@@ -1,10 +1,388 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <!-- Butoni ku mund te shkruajme nje koment per qdo panel te caktuar -->
+                    <div class="col-md-12">
+                        <div class="reply-button">
+                            <button type="button" class="btn-reply" data-toggle="dropdown">Reply comment
+                                <label for="text" class="glyphicon glyphicon-chevron-right" rel="tooltip" title="search"></label>
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.html]: qekjo for te labela svyn. munesh mi perdor me naj rast qeter psh mi shfrytzu me t kliku edhe teksti n checkbox, po qetu kurgjo

File: src/main/resources/templates/index.css [hunk 15]
```diff
@@ -0,0 +1,467 @@
...
+    padding-left: 3px;
+    background-color: #f5f5f5;
+    border-radius: 0px;
+}
+/*--------------------------------------------*\
+    Modal-repy (inside Notification's popup)
+\*--------------------------------------------*/
+
+.modal-reply {
+    height: 500px;
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: nuk ki nevoj per height fiks

File: src/main/resources/templates/index.html [hunk 16]
```diff
@@ -1,10 +1,388 @@
...
+                            </button>
+                            <ul class="dropdown-menu slider slider-reply">
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.html]: i ki shti vetit therr nkom to e bo qet san. ma mir qitja inputin krejt kohen, edhe bone 1 button comment a najsan. e thjeshton edhe hek pak kod

File: src/main/resources/templates/index.css (hunk 15 above)
Comment (rilindpozhegu) [inline src/main/resources/templates/index.css]: po kur nuk pi jap height fiks pem dalin jashta at'her senet mrena modulit

File: src/main/resources/templates/login.html [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+</head>
+<body>
+    <nav class="navbar navbar-default container-fluid">
+        <div class="navbar-header">
+            <div class="nav-logo">
+                <a href="http://localhost/forum/src/main/resources/templates/"><img src="logo-forum.png"></a>
+            </div>
+        </div>
+    </nav>
+    <div class="text-center">
```
Comment (GranitSahatqija) [inline src/main/resources/templates/login.html]: pshelli qeto elementet e login me 1 container qe ka kuptim, jo me div me class text-center. logon ski nevoj mi shti me div t veqant per mi kap stylen.

File: src/main/resources/templates/login.html [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+        <div class="logo"><img src="skenderbeu.jpg" class="login-photo">
+
+        </div>
+        <!-- Main Form -->
+        <div class="login-form-1">
```
Comment (GranitSahatqija) [inline src/main/resources/templates/login.html]: a ki form t 2-te t login? se emertimi nashta i keq pak

File: src/main/resources/templates/index.html (hunk 9 above)
Comment (rilindpozhegu) [inline src/main/resources/templates/index.html]: e Rregullova

File: src/main/resources/templates/index.css [hunk 19]
```diff
@@ -0,0 +1,467 @@
...
+/*--------------------*\
+      slide-cover
+\*--------------------*/
+
+.cover-photos-slider {
+    max-width: 100%;
+    height: 400px;
+    position: relative;
+}
+@media screen and (max-width: 450px) {
```
Comment (albizeka) [inline src/main/resources/templates/index.css]: le ka ni rresht tthat ndermjet klasave :P

File: src/main/resources/templates/index.css [hunk 20]
```diff
@@ -0,0 +1,467 @@
...
+\*-------------------------------------------*/
+
+.search-icon {
+    position: relative;
+    color: #555;
+    display: block;
+    padding-left: 15px;
+}
+@media screen and (max-width: 450px) {
+    .search-icon {
```
Comment (albizeka) [inline src/main/resources/templates/index.css]: qitu munesh veq padding-left-in mja bo fix mes me bo rewrite positionin colorin edhe displayen

File: src/main/resources/templates/index.html [hunk 21]
```diff
@@ -1,10 +1,388 @@
...
-<html xmlns:th="http://www.thymeleaf.org">
+<!DOCTYPE html>
+<html>
+
 <head>
-    <title>Getting Started: Serving Web Content</title>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Home</title>
+    <!-- bootstrap -->
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```
Comment (albizeka) [inline src/main/resources/templates/index.html]: perdori librarit lokale mes i perdor online

File: src/main/resources/templates/index.html [hunk 22]
```diff
@@ -1,10 +1,388 @@
...
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li><a href="#">Profile</a>
+                            </li>
+                            <li role="separator" class="divider"></li>
+                            <li><a href="#">Logout</a>
+                            </li>
+                            <li><a href="login.html">Login.html</a>
+                            </li>
+                            <li><a href="http://localhost/forum/src/main/resources/templates/signup.html">Signup</a>
```
Comment (albizeka) [inline src/main/resources/templates/index.html]: perderi sa i ki views-at me ni folder munesh mja bo veq a href="signup.html" ski nevoj mi shkru krejt pathin

File: src/main/resources/templates/login.html [hunk 23]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Login</title>
+    <!-- bootstrap -->
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+    <link href='http://fonts.googleapis.com/css?family=Varela+Round' rel='stylesheet' type='text/css'>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
```
Comment (GranitSahatqija) [inline src/main/resources/templates/login.html]: <meta> vijin meniher mas <title> tagit. 2 her e ki load fontawesome me versione t ndrryshme

File: src/main/resources/templates/login.css [hunk 24]
```diff
@@ -0,0 +1,159 @@
+html,
+body {
```
Comment (albizeka) [inline src/main/resources/templates/login.css]: nejt edhe ktu ndermjet klasave, tagave, id-ve etj len ka ni rresht hapsin

File: src/main/resources/templates/login.html [hunk 25]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Login</title>
+    <!-- bootstrap -->
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```
Comment (albizeka) [inline src/main/resources/templates/login.html]: Edhe ktu perdor skripta lokalisht

File: src/main/resources/templates/.picasaoriginals/.picasa.ini (hunk 6 above)
Comment (GranitSahatqija) [inline src/main/resources/templates/.picasaoriginals/.picasa.ini]: foto me extension .ini qetash pash

File: src/main/resources/templates/login.html [hunk 26]
```diff
@@ -0,0 +1,69 @@
...
+                        <div class="form-group login-group-checkbox">
+                            <input type="checkbox" id="lg_remember" name="lg_remember">
+                            <label for="lg_remember">remember</label>
+                        </div>
+                    </div>
+                    <button type="submit" class="login-button"><i class="fa fa-chevron-right"></i>
+                    </button>
+                </div>
+                <div class="login-form">
+                    <p>forgot your password? <a href="#">click here</a>
```
Comment (albizeka) [inline src/main/resources/templates/login.html]: forgot your password click here e ket pjesen tjeter msohu me capital case mi shkru

File: src/main/resources/templates/index.css [hunk 27]
```diff
@@ -0,0 +1,467 @@
...
+    height: 40px;
+    font-size: 18px;
+    padding: 10px 16px 10px 40px;
+    width: 450px;
+}
+@media screen and (max-width: 450px) {
+    .search-icon.addon-lg .form-control {
+        height: 40px;
+        font-size: 18px;
+        width: 300px;
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: fshije height edhe font-size, i ki definu ma heret. n media queries e shkrun stylen qe do me tu ndrru, jo t njejtin. btw qeky media query ti kap viq pajisjet nen 450px. organizo strukturen me perdor ma pak media, jo me shku shum specifike

File: src/main/resources/templates/index.css [hunk 28]
```diff
@@ -0,0 +1,467 @@
...
+    padding-right: 5px;
+    margin-bottom: 10px;
+}
+.logo-foum {
+    height: 70px;
+}
+.navbar-form {
+    margin-left: 10px;
+}
+@media screen and (max-width: 450px) {
```
Comment (GranitSahatqija) [inline src/main/resources/templates/index.css]: organizoj media queries ma mir. zakonisht shkojin n fun t fajllit. sdo me thon mi perserit psh max:width:450px 20 her, bone 1 edhe shtine stylen qe do me tu ndrru n ket rezolucion
